plugins/pdr_deterministic_plugin/tests/simulation_telemetry.py

```diff
@@ -339,7 +341,29 @@ def check_logs(config):
             break
     if len(lines) == 0:
         print("Could not find log file in " + str(location_logs_can_be))
-        return 1        
+        return 1
+
+    saved_files_tests = True
+    locations_telemetry_csv = "/opt/ufm/ufm_plugin_pdr_deterministic/datastore"
+    OUTPUT_FILE_FORMAT = "%Y_%m_%d_%H_%M_%S.csv"
+
+    for option_location in ['abs','delta']:
+        input_path_dir = Path(join(locations_telemetry_csv,option_location))
+        files = list(input_path_dir.glob("*.csv"))
+        
+        if len(files) == 0: # testing we have files there
+            print(f"There is no files in the datastore location:{join(locations_telemetry_csv,option_location)}")
+            saved_files_tests = False
+            continue
+        files.sort(key=lambda p: p.name)
+        latest_file = files[0].name
+        saved_time = datetime.strptime(latest_file,OUTPUT_FILE_FORMAT)
+        different_time = datetime.now() - saved_time
+        print_test_result(f"the latest file saved at {join(locations_telemetry_csv,option_location)} before test running, more than 5 minutes {different_time.total_seconds() > 300}",
+                          different_time.total_seconds() > 300, False)
+        if different_time.total_seconds() > 300:
```

Comment:
What do you test here?
How can you verify that you haven't flooded the disk?

Reply:
I tested that the file was created when the ci started and before. Meaning there is a new file.

I don't flood the disk because I call the clean_old_files function each iteration in the main_flow function.

Reply:
You need to check that you don't flood the disk, not assume it. You need to create a scenario in which you have ten files, and you don't exceed this number.

Reply:
added a test to make sure we don't flood the disk.